src/show/pages/private/MyProjects/index.js

```diff
@@ -44,12 +40,13 @@ const MyProjects = () => {
   return (
     <Fragment>
       <div className='h-full w-full'>
-        {myProjects.length > 0 ? (
+        {projects ? (
```

Comment:
Also I really like myProjects....
```suggestion
        {projects.length > 0 ? (
```

Reply:
biniuca